static/app/utils/formatters.tsx

```diff
@@ -148,70 +148,60 @@ export function getDuration(
 }
 
 /**
- * Returns a human readable exact duration.
+ * Translates seconds into human readable format of seconds, minutes, hours, days, and years
  *
- * e.g. 1 hour 25 minutes 15 seconds
```

Comment:
Let's keep this bit of the comment, it makes it kind of clear what the return is

Reply:
it'd be cool if we had some kind of strftime-esq way to describe what you're going to get back. "human readable" isn't a strong/specific enough description for me. Human readable could mean `a minute ago` which this doesn't do. 

The example `1 hour 25 minutes 15 seconds` is much better at tell me what to expect, but it is dependant on whether `abbreviation` is set, and the exactness seems to be dependant on `minDuration` now.

If i were refactoring this I'd shove the implementation into https://github.com/getsentry/sentry/blob/04b011447c5680ae4e489a890915c4fcd2fb260f/static/app/utils/duration/formatDuration.tsx#L61 and use two format strings to describe the abbreviation true/false cases. The `minDuration` is already handled by properly observing `precision` in that function.

Then the implementation for `getExactDuration` would be something small like:
```
function getExactDuration(seconds: number, abbreviation: boolean, minDuration: 'milliseconds' | 'seconds') {
  return formatDuration({
    precision: minDuration === 'milliseconds' ? 'ms' : 'sec',
    style: abbreviation ? 'human-absolute-abbr' : 'human-absolute', 
    duration: [seconds, 'sec']
  });
}
```